keyboards/install.php

```diff
@@ -281,21 +307,27 @@ protected static function WriteAndroidBoxes() {
       $referrer = "source=keyman&package={$u['id']}";
       if(!empty($u['bcp47'])) $referrer .= "&bcp47={$u['bcp47']}";
 
+      $downloadKeymanTitle = _m("download_keyman_title");
+      $withPlayStore = _m("with_play_store", $h['name']);
+      $installFromPlayStore = _m("install_from_play_store");
+      $installKeymanAndKeyboard = _m("keyman_and_keyboard_for_platform", $h['name'], "Android");
+      $alreadyInstalled = _m("already_installed");
+      $downloadJustKeyboard = _m("download_just_keyboard");
       $downloadKeymanUrl = PlayStore::url . "&referrer=" . rawurlencode($referrer);
+      $andThenInstallInApp = _m("and_then_install_in_the_app");
```

Comment:
I'm not a big fan of introducing lots of variables which results in repetition and divergence of variable names (e.g. `$andThenInstallInApp` vs `and_then_install_in_the_app`). Is there a way we just embed in the strings instead?

e.g. in Locale.php, add the following line to declare a global variable `$_m`:

```php
$_m = '_m';
```

```php
echo <<<END
  ... {$_m('already_installed')} ...
```

This ends up calling `_m(...)` which is kinda weird and kinda helpful.

Here and in #622, #604.

https://stackoverflow.com/a/43759648/1836776